lib/api/2.0/tags.js

```diff
@@ -8,6 +8,19 @@ var nodes = injector.get('Http.Services.Api.Nodes');
 var tags = injector.get('Http.Services.Api.Tags');
 var _ = injector.get('_'); // jshint ignore:line
 var Errors = injector.get('Errors');
+var parseUrl = require('url').parse;
+
+function getTagName(req) {
+    //TODO: sway 1.0.0 implements a fix for this, but it is lacking in the ^0.6.0 versions
+    //      so we duplicate the 1.0.0 implementation here. 
+    // Ref: https://github.com/apigee-127/sway/commit/1b7acd9743aae2f2f94057d3f8911e2af72614f5
```

Comment:
@yyscamper: swagger-express-mw includes swagger-node-runner includes sway.  We do not control the dependency list of swagger-node-runner.